gulpfile.js

```diff
@@ -327,6 +327,7 @@ gulp.task('protractor', ['webdriver_update'], function () {
       process.exit(0);
     })
     .on('error', function(err) {
+      console.log(err);
```

Comment:
we use consistently `console.log()`  on gulp/grunt. Should I change all of them to `.error` for consistency?